doc/nrf/libraries/networking/azure_iot_hub.rst

```diff
@@ -36,12 +38,17 @@ See `Creating an Azure IoT Hub instance using the Azure portal`_ for more inform
 The connection to Azure IoT Hub with MQTT is secured using TLS.
 For testing purposes, see `Creating Azure IoT Hub certificates`_ for the steps to create certificates and a private key for the leaf device, and to register the generated test root certificate to be used with an IoT hub.
 
-The Azure IoT Hub library requires the provisioning of the following certificates and a private key for a successful TLS connection:
+The Azure IoT Hub library requires the provisioning of the following certificates and private key for a successful TLS connection:
 
-1. `Baltimore CyberTrust Root certificate`_ - Server certificate, used to verify the server's certificate while connecting.
+1. `Baltimore CyberTrust Root Certificate`_ - Server certificate, used to verify the server's certificate while connecting.
```

Comment:
Shouldn't DigiCert Global Root G2 also be a requirement in this list?

Reply:
It's not required, just recommended right now: https://techcommunity.microsoft.com/t5/internet-of-things-blog/azure-iot-tls-critical-changes-are-almost-here-and-why-you/ba-p/2393169 ... but yes, it would be good to mention it. This is mentioned in line 47 below.

Reply:
Yes, but when Microsoft plan to stop using the Baltimore certificates in about a year, I think it would be smart to just tell everyone how they store both certificates on the device as the default procedure. Having it as a separate note makes no sense in my opinion, as our explanation will be less clear on what exact steps to do and it will be more likely some customers doesn't do it.

Reply:
I think the note stands out quite well since its marked `important`. I could expand on the note to include a recommendation that the second sec tag :kconfig:option:`CONFIG_AZURE_IOT_HUB_SECONDARY_SEC_TAG` should be populated with the baltimore certificates. If you think that is sufficient enough.

Reply:
Updated the note to include the recommendation.